spring-cloud-dataflow-rest-client/src/test/java/org/springframework/cloud/dataflow/rest/client/DataflowTemplateTests.java

```diff
@@ -107,6 +109,23 @@ public void testThatObjectMapperGetsPrepared() {
 		assertCorrectMixins(objectMapper);
 	}
 
+	@Test
+	public void testJobParameters() throws JsonProcessingException {
+		final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
I think you can remove the constuction (and prepare) of the OM here and just use `this.mapper`.